File: Modulo14/Funcionario-CSharp/Medico.cs
using System;

public class Medico : Funcionario {

    private string especialidade;

    public Medico(int CPF, string nome, string especialidade) : base(CPF, nome) {
        this.especialidade = especialidade;
    }

    override public string getProfissao() {
        return "Medico";
    }

    public string getEspecialidade() {
        return especialidade;
    }

    new public void imprime() {
        base.imprime();
        Console.WriteLine("Especialidade: {0}", this.especialidade);
    }
}

File: Modulo14/Funcionario/Medico.cpp
#include "Medico.h"

Medico::Medico(string CPF, string nome, string especialidade) :
Funcionario(CPF, nome), especialidade(especialidade) {
}

Medico::~Medico() {

}

string Medico::getProfissao() const {
    return "Médico";
}

void Medico::imprime() const {
    Funcionario::imprime();
    cout << "Especialidade : " << especialidade << endl;
}





